include/eve/detail/abi.hpp

```diff
@@ -7,6 +7,9 @@
 //==================================================================================================
 #pragma once
 
+// Faster than std::forward
+#define EVE_FWD(...) static_cast<decltype(__VA_ARGS__) &&>(__VA_ARGS__)
+
```

Comment:
Is there any reason it's variadic?

Reply:
So we can call on expression containing comma